### PR TITLE
Telescopic Cane and the loadout option

### PR DESCRIPTION
**Pull request by WilliamMurdoch**
Adds a telescopic cane (at the request of Casper/TheGreyWolf) and merged the white cane and crutches under the cane option in the loadout.

### CODE REVIEW

**Comment by WilliamMurdoch** [comment]
!review

**Comment by WilliamMurdoch** [comment]
To clear up, the edit made to the telebaton was just moving the sprite over by a few pixels to center it. Nothing major has been changed about it.

**Comment by Greenjoe12345** [comment]
I think it'll need to be adjusted so that if the the telescopic cane is retracted, it won't help you stand if you're missing a foot?
I don't know how to do that but it doesn't currently look like it does that

**Comment by WickedCybs** [comment]
I really do not think ten force is appropriate for a loadout item that can be collapsed to be small and fit into a pocket very easily. This is basically only a _slightly weaker_ telebaton at the moment. The tradeoff with something like the normal cane which also has 10 force, is that it's bulky and can't be so easily used as a weapon like that which is shared with similar 10 force items like the fire extinguisher and such.

Maybe reduce the active force to six or so.

File: html/changelogs/William Murdoch - Telecopic Baton.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Added a telecopic baton which can be found in the loadout under cane."
```
Comment (WickedCybs) [inline html/changelogs/William Murdoch - Telecopic Baton.yml:41]: Should probably reword this to "telescopic cane" instead of "telescopic baton" to avoid confusion as well.

Comment (WilliamMurdoch) [reply html/changelogs/William Murdoch - Telecopic Baton.yml:41]: Kept getting them confused in my head when I was writing it out. I'll fix that up and change the force down to a five.

**Comment by WilliamMurdoch** [comment]
> I think it'll need to be adjusted so that if the the telescopic cane is retracted, it won't help you stand if you're missing a foot?
> I don't know how to do that but it doesn't currently look like it does that

I don't entirely know how to do that either. I'll dig into it a bit more and see what I can do.

**Comment by Greenjoe12345** [comment]
since there's a can_support var now, can you add can_support =  false to the white cane?
since well, it's not meant to be the kinda cane you use to help walk, it's one a visually impaired person uses to not trip on stuff.

**Comment by WilliamMurdoch** [comment]
> since there's a can_support var now, can you add can_support = false to the white cane?
> since well, it's not meant to be the kinda cane you use to help walk, it's one a visually impaired person uses to not trip on stuff.

Added and such. It's a fair point although it makes me wonder if colorable canes would ever exist.

**Review by Alberyk — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Comment by WilliamMurdoch** [comment]
Made a mistake _again_. Gonna remake the pr in a bit.
